tests/php/bootstrap.php

```diff
@@ -100,6 +100,26 @@ function _manually_install_woocommerce() {
 	}
 }
 
+/**
+ * Replace Full Sync module with Full Sync Immediately
+ *
+ * @param array $modules Sync Modules.
+ *
+ * @return array
+ */
+function jetpack_full_sync_immediately_on( $modules ) {
+	foreach ( $modules as $key => $module ) {
+		if ( in_array( $module, array( 'Automattic\\Jetpack\\Sync\\Modules\\Full_Sync', 'Jetpack_Sync_Modules_Full_Sync' ), true ) ) {
```

Comment:
5 versions later, I'd assume we can clean the `LEGACY_SYNC_MODULES_MAP` up. Not necessarily in this PR though.

Reply:
yes! https://github.com/Automattic/jetpack/pull/13108

Reply:
You're the best!